var/spack/repos/builtin/packages/hal/package.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Hal(MakefilePackage):
+    """HAL is a structure to efficiently store and index multiple
+    genome alignments and ancestral reconstructions. HAL is a
+    graph-based representation which provides several advantages
+    over matrix/block-based formats such as MAF, such as
+    improved scalability and the ability to perform queries with
+    respect to an arbitrary reference or subtree."""
```

Comment:
The first paragraph in the docstring is used as a short description of the package in many places (e.g. module files), so I would suggest something like:
```suggestion
    """HAL is a structure to efficiently store and index multiple
    genome alignments and ancestral reconstructions. 
    
    HAL is a graph-based representation which provides several advantages
    over matrix/block-based formats such as MAF, such as
    improved scalability and the ability to perform queries with
    respect to an arbitrary reference or subtree."""
```